modules/nextMillenniumBidAdapter.js

```diff
@@ -291,6 +291,11 @@ export function getImp(bid, id, mediaTypes) {
     },
   };
 
+  const gpid = bid?.ortb2Imp?.ext?.gpid;
+  const pbadslot = bid?.ortb2Imp?.ext?.data?.pbadslot;
+  if (gpid) imp.ext.gpid = gpid;
+  if (pbadslot) imp.ext.data = { pbadslot };
```

Comment:
i recommend if(pbadslot && pbadslot !===gpid) or something

Reply:
If it's the same, why shouldn't we just pass both?

Reply:
Save a penguin

Reply:
Up to you though